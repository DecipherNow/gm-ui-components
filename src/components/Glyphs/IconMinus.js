import React from "react"
import Icon from "../Icon";

export default function Minus(props) {
  return (
    <Icon {...props} glyphName="Minus">
        <path d="M2.99999996,12 C2.99999996,11.4477152 3.44771521,11 3.99999996,11 L20,11 C20.5522848,11 21,11.4477152 21,12 C21,12.5522848 20.5522848,13 20,13 L3.99999996,13 C3.44771521,13 2.99999996,12.5522848 2.99999996,12 Z"   fillRule="nonzero" />

    </Icon>
  );
}

Minus.displayName = "Minus";
