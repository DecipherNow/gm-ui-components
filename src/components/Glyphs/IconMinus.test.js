import React from "react";
import { shallow } from "enzyme";
import IconMinus from "./IconMinus.svg";
describe("IconMinus", () => {
  it("matches snapshot", () => {
    
    const aIconMinus = shallow(<IconMinus />).find("IconMinus.svg").dive();
    console.log(aIconMinus.debug())
    expect(aIconMinus).toMatchSnapshot();
  });
});