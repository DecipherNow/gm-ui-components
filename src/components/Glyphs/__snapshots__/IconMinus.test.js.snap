// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconMinus matches snapshot 1`] = `<IconMock />`;
